### ISSUE TITLE
Install Atom package got compile tool error

### ISSUE THREAD

**Issue by rogerxu** [rogerxu/rogerxu.github.io#2]
It looks like something wrong with:

- HTTP proxy settings
- node version
- python version

**Comment by rogerxu** [comment]
It is issue related to corporate proxy, see atom/apm#322

**Comment by rogerxu** [comment]
**Windows temporary (cmd):**

```sh
set ATOM_ELECTRON_URL=http://gh-contractor-zcbenz.s3.amazonaws.com/atom-shell/dist`
```

**Windows permanently (cmd):**

```sh
setx ATOM_ELECTRON_URL http://gh-contractor-zcbenz.s3.amazonaws.com/atom-shell/dist /M
```

**Sh/Bash (Linux/Mac):**

```sh
export ATOM_ELECTRON_URL=http://gh-contractor-zcbenz.s3.amazonaws.com/atom-shell/dist
```

**Comment by rogerxu** [comment]
This issue has been fixed in Atom 1.10.0.
